src/displayapp/screens/Alarm.cpp

```diff
@@ -0,0 +1,251 @@
+/*  Copyright (C) 2021 mruss77, Florian
+
+    This file is part of InfiniTime.
+
+    InfiniTime is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    InfiniTime is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+#include "Alarm.h"
+#include "Screen.h"
+#include "Symbols.h"
+
+using namespace Pinetime::Applications::Screens;
+using Pinetime::Controllers::AlarmController;
+
+static void btnEventHandler(lv_obj_t* obj, lv_event_t event) {
+  Alarm* screen = static_cast<Alarm*>(obj->user_data);
+  screen->OnButtonEvent(obj, event);
+}
+
+Alarm::Alarm(DisplayApp* app, Controllers::AlarmController& alarmController)
+  : Screen(app), running {true}, alarmController {alarmController} {
+
+  time = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_font(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &jetbrains_mono_76);
+  lv_obj_set_style_local_text_color(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GRAY);
+
+  alarmHours = alarmController.Hours();
+  alarmMinutes = alarmController.Minutes();
+  lv_label_set_text_fmt(time, "%02lu:%02lu", alarmHours, alarmMinutes);
+
+  lv_obj_align(time, lv_scr_act(), LV_ALIGN_CENTER, 0, -25);
+
+  btnHoursUp = lv_btn_create(lv_scr_act(), nullptr);
+  btnHoursUp->user_data = this;
+  lv_obj_set_event_cb(btnHoursUp, btnEventHandler);
+  lv_obj_set_size(btnHoursUp, 60, 40);
+  lv_obj_align(btnHoursUp, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 20, -85);
+  txtHrUp = lv_label_create(btnHoursUp, nullptr);
+  lv_label_set_text(txtHrUp, "+");
+
+  btnHoursDown = lv_btn_create(lv_scr_act(), nullptr);
+  btnHoursDown->user_data = this;
+  lv_obj_set_event_cb(btnHoursDown, btnEventHandler);
+  lv_obj_set_size(btnHoursDown, 60, 40);
+  lv_obj_align(btnHoursDown, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 20, 35);
+  txtHrDown = lv_label_create(btnHoursDown, nullptr);
+  lv_label_set_text(txtHrDown, "-");
```

Comment:
All of these `lv_label_set_text` functions can be changed to `lv_label_set_text_static`

Reply:
Fixed